packages/cli/src/util/pruneOldFilesInDir.ts

```diff
@@ -1,17 +1,25 @@
 import fs from "node:fs";
 import path from "node:path";
 
-export function pruneOldFilesInDir(dirpath: string, maxAgeMs: number): void {
+export function pruneOldFilesInDir(dirpath: string, maxAgeMs: number): number {
+  let deletedFiles = 0;
   for (const entryName of fs.readdirSync(dirpath)) {
     const entryPath = path.join(dirpath, entryName);
 
     const stat = fs.statSync(entryPath);
     if (stat.isDirectory()) {
-      pruneOldFilesInDir(entryPath, maxAgeMs);
+      deletedFiles += pruneOldFilesInDir(entryPath, maxAgeMs);
     } else if (stat.isFile()) {
       if (Date.now() - stat.mtimeMs > maxAgeMs) {
         fs.unlinkSync(entryPath);
+        deletedFiles += 1;
       }
     }
   }
+
+  // if all files are deleted, delete the directory
```

Comment:
why do we need to delete the directory?

Reply:
otherwise there will be a lot of historical empty directories for nothing

Reply:
isn't this part of the beacon folder (`dataDir`)? But fair enough, we can just remove the folder as well